mne/io/base.py

```diff
@@ -1765,6 +1769,51 @@ def to_data_frame(self, picks=None, index=None,
                                default_index=['time'])
         return df
 
+    def describe(self):
+        """Describe channels."""
+        from scipy.stats import scoreatpercentile as q
+        nchan = self.info["nchan"]
+
+        # describe each channel
+        cols = defaultdict(list)
+        cols["name"] = self.ch_names
+        for i in range(nchan):
+            ch = self.info["chs"][i]
+            data = self[i][0]
+            cols["type"].append(channel_type(self.info, i))
+            cols["unit"].append(_unit2human[ch["unit"]])
+            cols["min"].append(np.min(data))
+            cols["q1"].append(q(data, 25))
+            cols["median"].append(np.median(data))
+            cols["q3"].append(q(data, 75))
+            cols["max"].append(np.max(data))
+
+        lens = {"name": max(4, max([len(_) for _ in cols["name"]])),
```

Comment:
`_` is usually for variables you don't use.

Reply:
Thanks @agramfort, I renamed the function to `si_prefix`, which I find even more specific for what it does. I've also changed the names in the list comps.